src/main_window.cpp

```diff
@@ -264,19 +264,48 @@ void MainWindow::installPkg()
     // Create game directory.
     auto gamesDirectory = readGamesDirectorySetting();
 
-    if (!QDir(gamesDirectory).mkdir(param.titleId())) {
-        QString msg(
-            "Cannot create directory %1 inside %2. "
-            "If you have a failed installation from a previous attempt, you will need to remove this directory before trying again.");
+    // Get Param information
+    auto appver = param.appver();
+    auto category = param.category();
+    auto shortContentId = param.shortContentId();
+    auto title = param.title();
+    auto titleID = param.titleId();
+
+    // Check if file is Patch/DLC.
+    bool PatchOrDLC = false;
+    if (category.startsWith("gp") || category.contains("ac")) {
+        PatchOrDLC = true;
+    }
 
-        QMessageBox::critical(&progress, "Error", msg.arg(param.titleId()).arg(gamesDirectory));
-        return;
+    // If the game exists and the file isn't a Patch/DLC, don't allow.
+    if (!QDir(gamesDirectory).mkdir(titleID) && !PatchOrDLC) {
```

Comment:
What if the user trying to install the game (not patch or DLC) but the directory cannot be created for some reasons?